common/static/common/js/karma.common.conf.js

```diff
@@ -235,8 +290,52 @@ var getConfig = function (config, useRequireJs) {
     };
 };
 
+var configure = function(data) {
+    var useRequireJs = data.useRequireJs === undefined ? true : useRequireJs,
+        baseConfig = getBaseConfig(data.config, useRequireJs);
+
+    var files = _.flatten(
+        _.map(
+            ['libraryFilesToInclude', 'libraryFiles', 'sourceFiles', 'specFiles', 'fixtureFiles', 'runFiles'],
+            function(item) { return data.files[item] || []; }
+        )
+    );
+
+    files.unshift(
+        {pattern: path.join(appRoot, 'common/static/common/js/jasmine.common.conf.js'), included: true}
+    );
+
+    if (useRequireJs) {
+      files.unshift({pattern: 'common/js/utils/require-serial.js', included: true});
+    }
+
+    files = setDefaults(files);
+
+    // With nocache=true, Karma always serves the latest files from disk.
+    // However, that prevents coverage tracking from working.
+    // So we only set it if coverage tracking is off.
+    setNocache(files, !data.config.coverage);
```

Comment:
@andy-armstrong If `nocache=false`, Karma reads files once and keeps serving them from its cache. So if I make a change to a file Karma will need to be restarted for the changes to be reflected. In dev mode, we want changes to be reflected on a page refresh. And in run mode the browser will request a file only once anyway. So as long as we do not need coverage we set it to `true`.